docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -91,15 +91,69 @@ for HTTP and WS.
 
 Authentication tokens expire 5 minutes after being generated. It is necessary to generate a new authentication 
 token if access is required after token expiration.     
+
+## JWT Public Key Authentication
+
+Enable authentication from the command line and supply the public key of the external JWT token. The JWT token must use the `RS256` algorithm.
+
+JWT public authentication disables the Besu `/login` endpoint, meaning [username and password authentication](username-and-password-authentication) will not work.
+
+### 1. Generate a Private and Public Key Pair
+
+The private and accompanying public key file must be in the `.pem` format.
+
+The key must use a RSA private key of at least 2048 bits.
+
+!!! example "Example using OpenSSL"
+    ```bash
+    openssl genrsa -out privateKey.pem 4096
```

Comment:
Should the example be consistent with the requirement of 2048 bits key? Here it generates a 4096 one.